spec/features/editing_file_attachments/edit_file_attachment_spec.rb

```diff
@@ -24,17 +24,17 @@ def and_i_edit_the_attachment_metadata
     stub_publishing_api_put_content(@edition.content_id, {})
     stub_asset_manager_receives_an_asset
 
-    @isbn = "9788700631625"
-    @unique_reference = "A unique reference"
+    unique_ref = "REF"
+    isbn = "9788700631625"
```

Comment:
They could be reversed, although it doesn't make any difference to me. I understand the principle here. It would difficult to uphold in general, so I'm going to leave it as-is.

Reply:
It's used here: https://github.com/alphagov/content-publisher/pull/1981/commits/81a3fb77624ffda49ed88801b7a0ce885ea3beaf#diff-c10aa9cc4442db3564ca175e62d23175R37